src/ResourceManager/Media/Commands.Media/MediaService/GetAzureRmMediaService.cs

```diff
@@ -0,0 +1,79 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Management.Media;
+using Microsoft.Azure.Commands.Media.Common;
+using Microsoft.Azure.Commands.Media.Models;
+
+namespace Microsoft.Azure.Commands.Media.MediaService
+{
+    /// <summary>
+    /// Get media service.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, MediaServiceNounStr), OutputType(typeof(PSMediaService))]
+    public class GetAzureRmMediaService : AzureMediaServiceCmdletBase
+    {
+        protected const string ResourceGroupParameterSet = "ResourceGroupParameterSet";
+        protected const string AccountNameParameterSet = "AccountNameParameterSet";
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ParameterSetName = ResourceGroupParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ParameterSetName = AccountNameParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The resource group name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ParameterSetName = AccountNameParameterSet,
+            HelpMessage = "The media service account name.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateLength(MediaServiceAccountNameMinLength, MediaServiceAccountNameMaxLength)]
+        [ValidatePattern(MediaServiceAccountNamePattern, Options = RegexOptions.None)]
+        [Alias(AccountNameAlias)]
```

Comment:
What do you mean by this? Should have a 'Name' as a alias to 'ResourceGroupName'?